src/DashJS/SDK/SDK.d.ts

```diff
@@ -0,0 +1,9 @@
+// @ts-ignore
```

Comment:
Does it make sense to include the type definition in the repo? As far as I remember the type definition file should be geberated by the typescript compile during the build included in the npm package